### DISCUSSION TITLE
Avalanche pre-consensus will bring near-instant transactions on eCash (XEC)

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6762] [General]
What is the current state of UW's eCash APIs?
Are we ready to get near-instant transactions when the new consensus mechanism will be available?
Recently I noticed just one-confirmation validator approval (to achieve finality) through blockchair explorer (previously was needed getting at least 6 confirmations).

https://avalanche.cash/

![1000093091](https://github.com/horizontalsystems/unstoppable-wallet-android/assets/113377431/ec23228e-a968-4aca-a5c1-361943e5ff92)

**Comment by abdrasulov** [comment]
In Unstoppable the confirmation for the transactions is already set to 1 block.
